src/shipit_code_coverage_backend/shipit_code_coverage_backend/services/active_data.py

```diff
@@ -29,15 +29,10 @@ def __init__(self):
         )
 
     async def __aenter__(self):
-        # TODO: Should we ping the server here (overkill ?)
         return self.client
 
     async def __aexit__(self, *args, **kwargs):
-
-        # TODO: Find out why this line fails on
-        # RuntimeWarning: coroutine 'ClientSession.close' was never awaited
-        # It should remove the warning about unclosed connector
-        pass
+        await self.client.transport.close()
```

Comment:
I wonder if there's a way to keep the connection open, so that we don't have to pay the connection price all the time we make a request.
aiohttp suggests having a single session with all the connections so that some resources are shared, but I can't find the docs anymore.